src/Keboola/DbExtractor/TableResultFormat/Table.php

```diff
@@ -64,7 +64,7 @@ public function setRowCount(?int $rowCount): self
 
     public function addColumn(TableColumn $column): self
     {
-        $this->columns[] = $column;
+        $this->columns[$column->getSanitizedName()] = $column;
```

Comment:
why not use `ordinalPosition` here?

Reply:
`ordinalPosition` is not require param

Reply:
@ondrajodas I think it should be,  all databases use it.

Reply:
@pivnicek impala and firebird don't use that:
https://github.com/keboola/db-extractor-impala/blob/master/src/Keboola/DbExtractor/Extractor/Impala.php#L236L240

Reply:
ah, right, only RDBs will have it.  I suppose we can control the ordering in the getTables method.  imo it's best to use ordinal position for that (when possible) to make sure it's the same order as the source table.

Reply:
ok, if when possible we use ordinal position - in last my commit